packages/postgraphile-core/__tests__/integration/schema/__snapshots__/disabledTagsNoLegacyRelations.test.js.snap

```diff
@@ -1628,1025 +1628,1025 @@ enum LeftArmsOrderBy {
+  """Deletes a single \`PersonSecret\` using its globally unique id."""
   deletePersonSecret(
-    \\"\\"\\"
+    """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
-    \\"\\"\\"
+    """
     input: DeletePersonSecretInput!
   ): DeletePersonSecretPayload
 
-  \\"\\"\\"Deletes a single \`PersonSecret\` using a unique key.\\"\\"\\"
+  """Deletes a single \`PersonSecret\` using a unique key."""
   deletePersonSecretByPersonId(
-    \\"\\"\\"
+    """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
-    \\"\\"\\"
+    """
     input: DeletePersonSecretByPersonIdInput!
   ): DeletePersonSecretPayload
   intSetMutation(
-    \\"\\"\\"
+    """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
-    \\"\\"\\"
+    """
     input: IntSetMutationInput!
   ): IntSetMutationPayload
   issue756Mutation(
-    \\"\\"\\"
+    """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
-    \\"\\"\\"
+    """
     input: Issue756MutationInput!
   ): Issue756MutationPayload
   issue756SetMutation(
-    \\"\\"\\"
+    """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
-    \\"\\"\\"
+    """
     input: Issue756SetMutationInput!
   ): Issue756SetMutationPayload
-  jsonIdentityMutation(
-    \\"\\"\\"
-    The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
-    \\"\\"\\"
-    input: JsonIdentityMutationInput!
-  ): JsonIdentityMutationPayload
   jsonbIdentityMutation(
-    \\"\\"\\"
+    """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
-    \\"\\"\\"
+    """
     input: JsonbIdentityMutationInput!
   ): JsonbIdentityMutationPayload
   jsonbIdentityMutationPlpgsql(
-    \\"\\"\\"
+    """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
-    \\"\\"\\"
+    """
     input: JsonbIdentityMutationPlpgsqlInput!
   ): JsonbIdentityMutationPlpgsqlPayload
   jsonbIdentityMutationPlpgsqlWithDefault(
-    \\"\\"\\"
+    """
     The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
-    \\"\\"\\"
+    """
     input: JsonbIdentityMutationPlpgsqlWithDefaultInput!
   ): JsonbIdentityMutationPlpgsqlWithDefaultPayload
+  jsonIdentityMutation(
+    """
+    The exclusive input argument for this mutation. An object type, make sure to see documentation for this object’s fields.
+    """
+    input: JsonIdentityMutationInput!
+  ): JsonIdentityMutationPayload
```

Comment:
This has been reordered I think?

EDIT: ah, it's because my old schema sorter and GraphQL's differ slightly. This is fine.